src/assets/translations/en/main.json

```diff
@@ -1418,8 +1418,10 @@
       },
       "connect": {
         "header": "Pair Ledger",
-        "body": "Click Pair then select your Ledger from the popup window",
-        "button": "Pair"
+        "pairNewDeviceBody": " Click 'Pair new device' to get started",
+        "pairNewDeviceButton": "Pair new device",
+        "pairExistingDeviceButton": "Pair existing device",
+        "pairExistingDeviceBody": "Connecting a different Ledger device to last time? Click 'Pair new device' to start fresh."
```

Comment:
Happy with this for the time being as an oversimplified copy, but we may want to improve it, as the assumption here terminology-wise is that device === seed, however a user may:
- use a temporary 25th word in their Ledger
- have a permanent 25th word associated to a password
- setup their Ledger again with a diff seed

@twblack88 may have some inputs on a better wording here

Reply:
Simpler is: "connecting a different ledger device? Click..."